Program.cs

```diff
@@ -2,60 +2,30 @@
 
 namespace objected_oriented_programming
 {
-   abstract class MOSGORGEOTREST
-    {
-        string ROLE { get; set; }
-        public abstract void Act();
-    }
-
-    class CLient : MOSGORGEOTREST
-    {
-        
-       public override void Act()
-        {
-            Console.WriteLine("Делаю заказа");
-        }
-
-    }
-
-    class TreatySection : MOSGORGEOTREST
-    {
-        public override void Act()
-        {
-            Console.WriteLine("Заключаю договор");
-        }
-    }
-
-    class TaskDepartment : MOSGORGEOTREST
-    {
-        public override void Act()
-        {
-            Console.WriteLine("Подготавливаю план работ!");
-        }
-    }
-
-    class Laboratory : MOSGORGEOTREST
-    {
-        public override void Act()
-        {
-            Console.WriteLine("Делаю лабораторные испытания!");
-        }
-    }
-
-    class FieldWork : MOSGORGEOTREST
-    {
-        public override void Act()
-        {
-            Console.WriteLine("Отбор проб из скважины!");
-        }
-
-    }
-
     class Program
     {
         static void Main(string[] args)
         {
-            
+            var point = new Point(10, 50, 1, false);
+            Console.WriteLine(point.ToString());
```

Comment:
ToString тут не нужен. Он и так будет вызван автоматически.
```suggestion
            Console.WriteLine(point);
```